HOS-23177: user profile is not present in ClientStats

chayan-04 opened the pull request:
We were facing issue in recieving user-profile
in telegraf since it uses dcd shared memory.
Fixed it now using telegraf helper exe.

## Summary
<!-- Mandatory
Explain here the why, the rationale and motivation, for the changes.
-->

## Checklist
<!-- Mandatory
Please confirm the following by replacing the space with an "x" between the []:
-->

- [ ] No AI generated code was used in this PR

## Related issues
<!-- Mandatory
All PRs should resolve an issue, if one does not exist, please open one.
-->

resolves #

chayan-04 committed:
HOS-23177: user profile is not present in ClientStats

We were facing issue in recieving user-profile
in telegraf since it uses dcd shared memory.
Fixed it now using telegraf helper exe.

arnelplim commented on plugins/inputs/ah_wireless_v2/ah_wireless_v2.go:
```diff
@@ -2626,7 +2629,7 @@ func Gather_Client_Stat(t *Ah_wireless, acc telegraf.Accumulator) error {
 			fields2["name"]				= strings.ReplaceAll(string(onesta.isi_name[:]), "\u0000", "")
 			fields2["host"]				= rt_sta.hostname
-//			fields2["profName"]			= "default-profile"			/* TBD (Needs shared memory of dcd)	*/
+			fields2["profName"]			= rt_sta.userprofile
```
Can you clarify what "shared memory of dcd" means?

chayan-04 replied:
@arnelplim , That data was supposed to be read from shared memory of dcd which is other C module on HOS. But we are retrieving in a different way now.

chayan-04 committed:
HOS-23203: Added bucket number to noise,snr,rssi

As an workaroud to overcome array sorting issue,
we are adding a bucket number in each bucket of
the nois, snr and rssi array.

chayan-04 committed:
HOS-23214: We need channel field in RfStats

Adding channel in rfstat.
